modules/storage/storage-account/version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
-  "version": "1.0",
+  "version": "2.0",
```

Comment:
will need to update the README to point for 2.0.1

Reply:
also need to regen, if you set this up, we can take care of it automatically for you.

Enable GitHub Worksflows in your fork to enable auto-generation of assets with our [GitHub Action](https://github.com/.github/workflows/push-auto-generate.yml).
To trigger GitHub Actions after auto-generation, [add a GitHub PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) as a secret in your forked repository called PAT.

Reply:
@AshutoshNirkhe , i poked the changes directly into your branch since everything else looks good.